CONTRIBUTING.md

```diff
@@ -33,32 +54,70 @@ You can browse our [issue tracker](https://github.com/aragon/aragon-cli/issues)
 
-* [Good first issue](https://github.com/aragon/aragon-cli/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) - issues which should only require a few lines of code, and a test or two.
+- [Good first issue](https://github.com/aragon/aragon-cli/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) - issues which should only require a few lines of code, and a test or two.
 
 ## Getting help
-If you need help, please reach out to Aragon core contributors and community members in the [#dev-help channel](https://aragon.chat/channel/dev-help) on the Aragon Chat.  We'd love to hear from you and know what you're working on!
 
-## Styleguides
+If you need help, please reach out to Aragon core contributors and community members in the [#dev-help channel](https://aragon.chat/channel/dev-help) on the Aragon Chat. We'd love to hear from you and know what you're working on!
 
-### Commits
+## Style Guide
 
-* Use the present tense ("Add feature" not "Added feature")
-* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
-* Limit the first line to 72 characters or less
-* Reference issues and pull requests liberally after the first line
+[Prettier](https://prettier.io) will catch most styling issues that may exist in your code.
 
->A good rule of thumb is that your commit message follows these rules if you can prefix it with "This commit will ..." and it makes sense.
+However, there are still some styles that Prettier cannot pick up.
 
->For example, "This commit will **add a feature**" vs "This commit will **added a feature**".
+## Semantic Commit Messages
 
-### JavaScript
+See how a minor change to your commit message style can make you a better programmer.
+
+Format: `<type>(<scope>): <subject>`
+
+`<scope>` is optional
+
+### Example
+
+```
+feat: allow overriding of webpack config
+^--^  ^------------^
+|     |
+|     +-> Summary in present tense.
+|
++-------> Type: chore, docs, feat, fix, refactor, style, or test.
+```
+
+The various types of commits:
+
+- `feat`: (new feature for the user, not a new feature for build script)
+- `fix`: (bug fix for the user, not a fix to a build script)
+- `docs`: (changes to the documentation)
+- `style`: (formatting, missing semi colons, etc; no production code change)
+- `refactor`: (refactoring production code, eg. renaming a variable)
+- `test`: (adding missing tests, refactoring tests; no production code change)
+- `chore`: (updating grunt tasks etc; no production code change)
+
+### Rules
+
+- Use lower case not title case!
+- Use the present tense ("Add feature" not "Added feature")
+- Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+- Limit the first line to 72 characters or less
+- Reference issues and pull requests liberally after the first line
+
+## JavaScript
 
 All JavaScript must adhere to [JavaScript Standard Style](https://standardjs.com/).
```

Comment:
I'm not really sure this is true anymore, since the project uses prettier. I'd just say ESLint and prettier are set up to automatically lint and format all code.